### DISCUSSION TITLE
Missing code

### DISCUSSION THREAD

**Discussion by Urseen** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#412] [Q&A]
When I try to edit the code of my vaccum-card, I only get 7 lines of code. Did I do something wrong? Where is the code for all the tiles and buttons?
Thanks in advance for all the help....

<img width="974" alt="Bildschirmfoto 2023-03-05 um 21 35 56" src="https://user-images.githubusercontent.com/124264375/222984603-e11725d7-ea4e-4962-a600-95440a319451.png">

**Comment by PiotrMachowski** [comment]
Card generates it internally to keep your config clean and make it possible to automatically apply improvements in the next versions. If you really want to make your config static you can click "Generate static config" button in the visual editor

**Reply by Urseen** [reply]
Solved, thank you!
